src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2808,4 +2808,19 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_to_record" xml:space="preserve">
     <value>Convert to record</value>
   </data>
+  <data name="Introduce_parameter" xml:space="preserve">
+    <value>Introduce parameter</value>
+  </data>
+  <data name="Introduce_parameter_for_0" xml:space="preserve">
+    <value>Introduce parameter for '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_for_all_occurrences_of_0" xml:space="preserve">
+    <value>Introduce parameter for all occurrences of '{0}'</value>
+  </data>
+  <data name="Introduce_parameter_and_trampoline_for_0" xml:space="preserve">
+    <value>Introduce parameter and trampoline for '{0}'</value>
```

Comment:
i dont' like this... but i don't have any idea what woudl be better.  is this string not too long for the lightbulb as well?  can you show a picture?

Reply:
while we used the word `trampoline` in the design meeting I do not know if that is descriptive enough.

I would propose "Introduce parameter in new overload"

Reply:
or "new helper".  But i agree with Jon that "trampoline" is not really appropriate here.

Reply:
![image](https://user-images.githubusercontent.com/40616383/108760480-91b6e900-7502-11eb-9b94-c69413e4d041.png)

Reply:
Introduce new parameter overload for '{0}'? Trampoline is just adding an overload with new parameter set and calling old one, so using overload might be better for familiarity in terms for programmers.

Reply:
CHanged